reverse_proxy/molecule/default/verify.yml

```diff
@@ -155,6 +155,38 @@
       fail_msg: "GET queries should fail"
       success_msg: "GET query was sucessfully blocked"
 
+
+######################################################################
+# bioconductor tests
+
+  - name: test connection to /
+    uri:
+      url: http://bioconductor.{{zone}}/
+      force: true # avoid hitting cache
+    failed_when: false
+    changed_when: false
+    register: bioconductor_curl_root
+
+  - name: test connection to /
+    assert:
+      that: bioconductor_curl_root.status == 403
+      fail_msg: "bioconductor mirror requests without file extensions should fail"
+      success_msg: "bioconductor request without file extension was successfully blocked"
+
+  - name: test connection to packages/3.20/bioc/src/contrib/PACKAGES
+    uri:
+      url: http://bioconductor.{{zone}}/packages/3.20/bioc/src/contrib/PACKAGES
+      force: true  # avoid hitting cache
+    failed_when: false
+    changed_when: false
+    register: bioconductor_curl_packages
+
+  - name: test connection to /packages/3.20/bioc/src/contrib/PACKAGES
+    assert:
+      that: bioconductor_curl_packages.status == 200
+      fail_msg: "Request to valid file extension was blocked."
+      success_msg: "Request to valid file extension was not blocked."
+
```

Comment:
I think I've convinced myself the regex is fine. To ensure no-one incorrectly changes it in future would you mind adding a couple of tests to assert that data and annotation packages are blocked?
E.g. picking a couple of random annotations and experiments:
https://bioconductor.org/packages/3.20/data/annotation/src/contrib/chicken.db_3.13.0.tar.gz
https://bioconductor.org/packages/release/data/experiment/src/contrib/colonCA_1.46.0.tar.gz

Reply:
Added those :)